docker/fleet-os-3/download_toolchain.sh

```
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+#
+# This script does not work. Because of gitlab...
+#
+echo "DOES NOT WORK. DOWNLOAD MANUALLY BY CLICKING URI BELOW!" >&2
+echo "THEN UNZIP/UNTAR and copy toolchain as toolchain/oecore.sh" >&2
+
+set -e
+
+GITLAB_TOKEN="$1"
+SCRIPT_DIR="$(dirname $(realpath $0))"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)